packages/webapp/src/components/App.tsx

```diff
@@ -75,6 +75,24 @@ export default observer(function App() {
 		setSelectedKey('Dashboard')
 	}, [setSelectedKey])
 
+	const renderRepoMessageBar = () => {
```

Comment:
Why not just move RepoMessage bar into its own component?

Reply:
chose to go for this for now since we don't use this anywhere else. We'll definitely convert this to a component if needed.